4.analyze_data/notebooks/UMAP/nbconverted/0.UMAP.py

```diff
@@ -23,16 +23,17 @@
 umap_n_components = 2
```

Comment:
Looking at the line above, I think Way lab convention is a random seed of 0. Might consider changing but not a big deal

Reply:
Ah I didn't know that! In this case, for the context of PR, I am just rerunning to make sure that the original plot created with this random seed is replicated.

I think I will keep this in mind for a refactor in the future, so thank you for bringing it up!